apps/docs/pages/guides/database/connecting-to-postgres.mdx

```diff
@@ -8,82 +9,76 @@ export const meta = {
+2. Click `Database`.
+3. Find your Connection Info and Connection String. Direct connections are on port `5432`.
 
 <video width="99%" muted playsInline controls={true}>
   <source
-    src="https://xguihxuzqibwxjnimxev.supabase.co/storage/v1/object/public/videos/docs/api/api-url-and-key.mp4"
+    src="https://xguihxuzqibwxjnimxev.supabase.co/storage/v1/object/public/videos/docs/postgres-connection.mp4"
     type="video/mp4"
   />
 </video>
 
-## Direct connections
+## Connection Pooler
 
-Every Supabase project provides a full Postgres database. You can connect to the database using any tool which supports Postgres.
-
-### Finding your connection string
+Every Supabase project comes with PgBouncer for connection pooling. A connection pooler is useful for managing a large number of _temporary_ connections. For example, if you are using [Prisma](/docs/guides/integrations/prisma), Drizzle, Kysely, or anything deployed to a Serverless environment (AWS Lambdas or Edge Functions). You can find the connection pool config in the [Database settings](https://app.supabase.com/project/_/settings/database) inside the dashboard:
 
 1. Go to the `Settings` section.
 2. Click `Database`.
-3. Find your Connection Info and Connection String. Direct connections are on port `5432`.
+3. Find your Connection Info and Connection String. Connection pooling is on port `6543`.
 
 <video width="99%" muted playsInline controls={true}>
   <source
-    src="https://xguihxuzqibwxjnimxev.supabase.co/storage/v1/object/public/videos/docs/postgres-connection.mp4"
+    src="https://xguihxuzqibwxjnimxev.supabase.co/storage/v1/object/public/videos/docs/connection-pool-config.mp4"
     type="video/mp4"
   />
 </video>
 
-## Connection Pool
+## Choosing a connection method
+
+- The Serverless APIs provide programmatic access and have built-in connection pooling. You can use these for all browser and application interactions. We recommend using these wherever possible.
+- A "direct connection" is Postgres' native connection system. You should use this for tools which are always alive - usually installed on a long-running server, like Node.js, Ruby, Python, etc.
+- A "connection pooler" is a tool which keeps connections "alive". You should use this for serverless functions and tools which disconnect from the database frequently, like Prisma, Drizzle, Kysely, etc.
+
+Why would you use a connection pool? Primarily because the way that Postgres handles connections isn't very scalable for a large number of _temporary_ connections. You can use these simple questions to determine which connection method to use:
+
+- Are you connecting to a database and _maintaining_ a connection? If yes, use a direct connection.
+- Are you connecting to your database and then _disconnecting_ immediately (e.g. a serverless environment)? If yes, use a connection pool.
+
+## Connecting with SSL
+
+You should connect to your database using SSL wherever possible, to prevent snooping and man-in-the-middle attacks.
```

Comment:
Is it possible to connect to our instances without SSL?

Reply:
yeah I think it's still technically possible but not recommended